### ISSUE TITLE
[Bug]: On mobile once scrolled to the bottom, when I try scroll up it jumps back.

### ISSUE THREAD

**Issue by XeonG** [nextcloud/server#49045]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Latest chrome, on mobile once scrolled to the bottom, when I try scroll up it jumps back.

[Nextcloud Hub 9]() (30.0.1)

### Steps to reproduce

1. Goto cloud folder via web rbwoser, scroll to the bottom of a folder like one with pictures thumbs view
2. try to scroll up, get retarded behavior where it keeps jump the scroll position back down
3. that is it.. totally broken

### Expected behavior

scrolling works going up or down whenever I swipe the screen... shouldn't be hard, 

### Nextcloud Server version

30

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Updated from a MINOR version (ex. 28.0.1 to 28.0.2)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [x] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

 ./occ config:list system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.mustexist.me"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "30.0.1.2",
        "overwrite.cli.url": "*** WHY ISn't this removed VALUE***",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "app_install_overwrite": [
            "workflow_media_converter",
            "epubreader",
            "folderplayer",
            "files_reader"
        ],
        "maintenance": false,
        "updater.secret": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "loglevel": 2
    }
}

### List of activated Apps

Enabled:
  - activity: 3.0.0
  - app_api: 4.0.0
  - audioplayer: 3.4.1
  - bruteforcesettings: 3.0.0
  - circles: 30.0.0-dev
  - cloud_federation_api: 1.13.0
  - comments: 1.20.1
  - contactsinteraction: 1.11.0
  - dashboard: 7.10.0
  - dav: 1.31.1
  - federatedfilesharing: 1.20.0
  - federation: 1.20.0
  - files: 2.2.0
  - files_downloadlimit: 3.0.0
  - files_pdfviewer: 3.0.0
  - files_reminders: 1.3.0
  - files_sharing: 1.22.0
  - files_trashbin: 1.20.1
  - files_versions: 1.23.0
  - firstrunwizard: 3.0.0
  - logreader: 3.0.0
  - lookup_server_connector: 1.18.0
  - music: 2.0.1
  - nextcloud_announcements: 2.0.0
  - notifications: 3.0.0
  - oauth2: 1.18.1
  - password_policy: 2.0.0
  - photos: 3.0.2
  - privacy: 2.0.0
  - provisioning_api: 1.20.0
  - qownnotesapi: 24.9.0
  - recommendations: 3.0.0
  - related_resources: 1.5.0
  - serverinfo: 2.0.0
  - settings: 1.13.0
  - sharebymail: 1.20.0
  - snappymail: 2.38.2
  - support: 2.0.0
  - survey_client: 2.0.0
  - systemtags: 1.20.0
  - text: 4.1.0
  - theming: 2.5.0
  - twofactor_backupcodes: 1.19.0
  - updatenotification: 1.20.0
  - user_status: 1.10.0
  - viewer: 3.0.0
  - weather_status: 1.10.0
  - webhook_listeners: 1.1.0-dev
  - workflowengine: 2.12.0
Disabled:
  - admin_audit: 1.20.0
  - camerarawpreviews: 0.8.5 (installed 0.8.5)
  - encryption: 2.18.0
  - epubviewer: 1.6.3 (installed 1.6.3)
  - files_external: 1.22.0
  - files_rightclick: 0.15.1 (installed 1.6.0)
  - gptfreeprompt: 1.0.2 (installed 1.0.2)
  - maps: 1.4.0 (installed 1.4.0)
  - news: 24.0.0 (installed 24.0.0)
  - suspicious_login: 8.0.0
  - twofactor_nextcloud_notification: 4.0.0
  - twofactor_totp: 12.0.0-dev
  - user_ldap: 1.21.0
  - video_converter: 1.0.6 (installed 1.0.6)
  - workflow_media_converter: 1.11.7 (installed 1.11.7)

**Comment by joshtrichards** [comment]
> that is it.. totally broken

It does seem to scroll upward a little strangely, but only on the initial load (then it smooths out). Also, I can still scroll up or down.

Are you saying you can't scroll back upward at all?

Btw, This is in the list view with the smaller thumbnail. With the larger thumbnail view, it scrolls consistently (at least for me).

**Comment by XeonG** [comment]
Yeah it happens on mobile view with thumbnails, seems to happen all the time, doesn't happen on desktop though.. and also yes soon as I scroll to the bottom it won't let me scroll back up at all, just jumps back down.. basically unusable, unless I make sure I never hit the bottom of the scroll view.

**Comment by skjnldsv** [comment]
Can't reproduce anymore.
Pretty sure the new Files app fixed it. Please upgrade to latest supported NC versions (30 and above at the time of this comment)

**Comment by sahimjbc** [comment]
I am still facing this issue on nextcloud version 31.0.10.2
